index.html

```diff
@@ -1,108 +1,127 @@
-        </div>
-        <div class="email">
-            <input type="email" id="email" name="email" placeholder="Enter your email address">
-        </div>
-        <div class="button1">
-            <button>GET STARTED</button>
-        </div>
-        <div class="logo">
-          <i class="fa-brands fa-free-code-camp"></i>
-          <h2>Premium Materials</h2>
-          <p>Our trombones use the shiniest brass which is sourced locally. This will increase the longevity of your purchase.</p>
-        </div>
-        <div class="logo">
-          <i class="fa-solid fa-truck"></i>
-          <h2>Fast Shipping</h2>
-          <p>We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.</p>
-        </div>
-
-        <div class="logo">
-          <i class="fa-solid fa-battery-full"></i>
-          <h2>Quality Assurance</h2>
-          <p>For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.</p>
-        </div>
-        <div class ="Video">
-          
-        </div>
-        <iframe width="560" height="315" src="https://www.youtube.com/embed/y8Yv4pnO7qc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
-    
-        <div class="container">
-         <h3>Tenor trombones</h3>
-         <h3>$600</h3>
-         <p>Lorem ipsum.</p>
-         <p>Lorem ipsum.</p>
-         <p>Lorem ipsum.dolor</p>
-         <p>Lorem ipsum.</p>
-         <div class="button">
-          <button>SELECT</button>
+      <div class="title">
+        <h2>Handcrafted, home-made masterpieces</h2>
+      </div>
+      <div class="email">
+        <input
+          type="email"
+          id="email"
+          name="email"
+          placeholder="Enter your email address"
+        />
+      </div>
+      <div class="button1">
+        <button>GET STARTED</button>
+      </div>
+      <div class="logo">
+        <i class="fa-brands fa-free-code-camp"></i>
+        <h2>Premium Materials</h2>
+        <p>
+          Our trombones use the shiniest brass which is sourced locally. This
+          will increase the longevity of your purchase.
+        </p>
+      </div>
+      <div class="logo">
+        <i class="fa-solid fa-truck"></i>
+        <h2>Fast Shipping</h2>
+        <p>
+          We make sure you recieve your trombone as soon as we have finished
+          making it. We also provide free returns if you are not satisfied.
+        </p>
       </div>
 
+      <div class="logo">
+        <i class="fa-solid fa-battery-full"></i>
+        <h2>Quality Assurance</h2>
+        <p>
+          For every purchase you make, we will ensure there are no damages or
+          faults and we will check and test the pitch of your instrument.
+        </p>
+      </div>
+      <div class="Video"></div>
+      <iframe
```

Comment:
Please try as much as possible to avoid using inline styling as they don't separate content from design. Use your style.css file if you want to style your website.